lib/plugins/scoreboard.js

```diff
@@ -57,6 +58,47 @@ function inject (bot) {
     }
   })
 
+  bot.on('teams', (packet) => {
```

Comment:
I'd rather keep it separate from the `scoreboard.js`

Reply:
The vanilla client stores team and scoreboard data in the same class, but we can just make a teams.js

Reply:
Shouldn't this be `bot._client.on('scoreboard_team', (packet) => {`?

Reply:
No? That's just not the name of the packet:
https://github.com/PrismarineJS/minecraft-data/blob/master/data/pc/1.12.2/protocol.json#L2593

Reply:
oh you it does need to be `bot._client` and not just `bot` though